app/server/utils/metrics/metrics_const.py

```diff
@@ -71,3 +71,9 @@
 
 # Misc Values
 GROUPED = 'grouped'
+
+# Metric types
+COUNT = 'count'
+COUNT_AVERAGE = 'count_average'
+CURRENCY = 'currency'
+VALUE_TYPES = [COUNT, COUNT_AVERAGE, CURRENCY]
```

Comment:
👍

Reply:
don't love string comparison on frontend but not sure of a better alternative

Reply:
Still a string comparison, but I added PAYMENT to `constants.js` so it's a bit cleaner now